apps/api/src/app/events/e2e/trigger-event.e2e.ts

```diff
@@ -3223,10 +3226,169 @@ describe(`Trigger event - /v1/events/trigger (POST)`, function () {
       tenant,
       actor,
     };
-    console.log('request111', JSON.stringify(request, null, 2));
 
     return (await novuClient.trigger(request)).result;
   }
+
+  describe('Trigger Event New Dashboard - /v1/events/trigger (POST)', function () {
```

Comment:
should we rename it to v2 workflow?

```suggestion
  describe('Trigger Event v2 workflow - /v1/events/trigger (POST)', function () {
```

Reply:
yes